### PR TITLE
Support for UD_OLD_FRENCH added

### PR DESCRIPTION
**Pull request by Weyaaron**
I'm not  sure if additional formating is needed, this loads the corpus without additional reformatings of the source files.

### CODE REVIEW

File: flair/datasets/treebanks.py [hunk 1]
```diff
@@ -1303,3 +1306,30 @@ def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
...
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        web_path = "https://github.com/UniversalDependencies/UD_Old_French-SRCMF/tree/master"
```
Comment (alanakbik) [inline flair/datasets/treebanks.py:1326]: This does not download the correct file. Please check the downloaded file if it looks correct.

**Review by alanakbik — CHANGES_REQUESTED** [review]
This code does not run. If I do: 

```python
# load corpus (throws error)
corpus = UD_OLD_FRENCH()

# print corpus
print(corpus)

# print some sentences in corpus
print(corpus.train[1])
print(corpus.dev[1])
print(corpus.test[1])

# print UPOS tags
dictionary = corpus.make_tag_dictionary('upos')
print(dictionary)
```

It throws an error. Please check if the code runs locally before doing a PR!

File: flair/datasets/treebanks.py (hunk 1 above)
Comment (Weyaaron) [reply flair/datasets/treebanks.py]: The urls are fixed with the new commits added.

**Comment by alanakbik** [comment]
@Weyaaron thanks for adding this!

**Pull request merged**